### PR TITLE
Added utility function backend_get_qubit_params

### PR DESCRIPTION
**Pull request by zlatko-minev**
### Summary

Added utility to return all relevant single-qubit parameters as a pandas DataFrame object, which can easily be manipulated, sorted, queried.

### Details and comments

Example use 
```python 
### Standard import 
from qiskit import IBMQ
IBMQ.load_accounts()
backend = IBMQ.get_backend('ibmq_16_melbourne')

### Example use 
df = backend_get_qubit_params_df(backend)
df.sort_values('T2', ascending=False).style.set_precision(3)
```

Example output of qubit data sorted by T2:
image

### CODE REVIEW

**Comment by nonhermitian** [comment]
So, all of this info is already in tabular form in the jupyter backend monitor. The he exception is the ability to sort. But that could be added as a selector in the widget panel that holds the table.

**Comment by ajavadia** [comment]
yes if the intention is to be able to inspect this data (not necessarily do heavy post-analysis via Pandas dataframes), then the Jupyter widget is probably a quicker solution.
Pandas is currently not a dependency of Qiskit.

File: qiskit/tools/monitor/backend_overview.py [hunk 1]
```diff
@@ -195,3 +195,46 @@ def backend_overview():
...
+        for prop in props['qubits'][qnum]:
+            # T1, T2, freq, ro_err
+            RES[qnum][prop['name']] = prop['value']
+            # Gate error -- assumes structure is u1, u2, u3 first for all single q gates
+            gates = props['gates'][3*qnum:3*qnum+3]
+            for gate in gates:
+                RES[qnum][gate['gate']+'_err'] = gate['parameters'][0]['value']
+
+    # Format dataframe
+    df = pd.DataFrame(RES).transpose()
```
Comment (ajavadia) [inline qiskit/tools/monitor/backend_overview.py]: I don't actually see pandas (`pd`) imported anywhere, which is probably a bug. This points to the need for adding tests if new functionality is added.

**Comment by ajavadia** [comment]
I'll close this for now as I think the Jupyter backend monitor widget accomplishes this task. Feel free to reopen if you still think this is needed.

**Pull request closed without merging**